crates/primitives/src/snapshot/segment.rs

```diff
@@ -113,33 +115,30 @@ impl SnapshotSegment {
     /// # Note
     /// This function is tightly coupled with the naming convention defined in [`Self::filename`].
     /// Any changes in the filename format in `filename` should be reflected here.
-    pub fn parse_filename(
-        name: &OsStr,
-    ) -> Option<(Self, RangeInclusive<BlockNumber>, RangeInclusive<TxNumber>)> {
+    pub fn parse_filename(name: &OsStr) -> Option<(Self, RangeInclusive<BlockNumber>)> {
```

Comment:
any reason we use `OsStr`? this type to me is usually only for ffi so i'd rather not have to deal with it outside of that

Reply:
no need for it yeah